public/admin/config.yml

```diff
@@ -218,3 +218,22 @@ collections:
                 fields:
                   - { label: Question, name: question, widget: text }
                   - { label: Answer, name: answer, widget: markdown }
+  - name: data-disclaimers
+    label: Data disclaimers
+    format: json
+    description: Data disclaimers
+    slug: '{{slug}}'
+    editor:
+      preview: true
+    files:
+      - name: Disclaimers
+        label: Disclaimers
+        file: src/cms-content/data-disclaimers/disclaimers.json
+        fields:
+          - label: Disclaimers
+            name: disclaimers
+            widget: list
+            fields:
+              - { label: Disclaimer copy, name: disclaimerCopy, widget: markdown}
+              - { label: State codes, name: stateCodes, widget: string, required: false}
+              - { label: Fips codes, name: fipsCodes, widget: string, required: false}
```

Comment:
Consider using only FIPS codes to simplify the code. Also, consider adding a `comment` for each field explaining what it does (for states, will the disclaimer show up only on the state page or also for counties?). See https://www.netlifycms.org/docs/configuration-options/#fields